src/ty_module/instantiating_binders.md

```diff
@@ -138,8 +138,8 @@ As a concrete example, accessing the signature of a function we are type checkin
 [ch_placeholders_universes]: ../borrow_check/region_inference/placeholders_and_universes.md
 [`instantiate_binder_with_fresh_vars`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_trait_selection/infer/struct.InferCtxt.html#method.instantiate_binder_with_fresh_vars
 [`InferCtxt`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_trait_selection/infer/struct.InferCtxt.html
-[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.EarlyBinder.html
+[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/stable_mir/ty/struct.EarlyBinder.html
```

Comment:
don't link to Stable MIR plz, its `EarlyBinder` != `rustc_middle::ty::EarlyBinder` as used in the actual compiler, its but a copy.

```suggestion
[`EarlyBinder`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/type.EarlyBinder.html
```